stable/registry/cert/tls.key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Is it a testing purpose private key?

Reply:
This is a self-signed certificate that is used by default if the user doesn't provide a custom certificate. You could call it a testing purpose self-signed certificate, because it won't be used in production.